gradle.properties

```diff
@@ -2,14 +2,18 @@
 org.gradle.jvmargs=-Xmx1G
 # Fabric Properties
 # check these on https://fabricmc.net/use
-minecraft_version=1.17.1
-yarn_mappings=1.17.1+build.61
-loader_version=0.11.7
+minecraft_version=1.20.4
+yarn_mappings=1.20.4+build.3
+loader_version=0.15.7
+
 # Mod Properties
 mod_version=2.0.0
 maven_group=net.fabricmc
 archives_base_name=fabric-language-scala
 description=Fabric language module for Scala
 url=https://github.com/FabricMC/fabric-language-scala/
-# Other
-scala_version=3.0.2
+
+# Scala
+scala_version=3.4.2-RC1-bin-20240226-e0cb1e7-NIGHTLY
```

Comment:
Can we use a stable release?

Reply:
oh yea intellij wants you to use the latest build its weird but i can change it so its on a stable release